commit.go

```diff
@@ -166,6 +191,17 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
 			logrus.Debugf("error destroying signature policy context: %v", err2)
 		}
 	}()
+
+	// Check if we'd be allowed to run an image from the destination location.
+	allowed, err := policyContext.IsRunningImageAllowed(ctx, &referenceOnlyImage{dest: dest})
+	if err != nil {
+		return "", nil, "", errors.Wrapf(err, "running image with reference %q would not be allowed by policy, denying commit", transports.ImageName(dest))
+	}
+	if !allowed {
+		return "", nil, "", errors.Errorf("running image with reference %q would not be allowed by policy, denying commit", transports.ImageName(dest))
+	}
+	logrus.Debugf("running image with reference %q would be allowed by policy", transports.ImageName(dest))
```

Comment:
nit: 

logrus.Debugf("running image with reference %q is allowed by policy", transports.ImageName(dest))

Reply:
Sure, changing.